pyquil/operator_estimation.py

```diff
@@ -7,7 +7,8 @@
 import warnings
 from json import JSONEncoder
 from operator import mul
-from typing import List, Union, Iterable, Tuple, Optional, Dict
+from typing import List, Union, Iterable, Tuple, Optional, Dict, Callable
+from tqdm import tqdm
```

Comment:
let's drop the dependence on `tqdm` for now -- we can discuss adding it in another PR. one thing I like about pyquil is how we managed to prevent a proliferation of superfluous dependencies :)